example.yml

```diff
@@ -0,0 +1,21 @@
+name: Flowzone
```

Comment:
Could keep this example here, but I think it makes more sense for it to exist in the flowzone repo. Otherwise, we would need to update this repo anytime the "base" configuration changes.